packages/extras/src/lib/audio/AudioListener/AudioListener.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import { Three, useThrelteAudio } from '@threlte/core'
+  import { onDestroy } from 'svelte'
+  import { AudioListener as ThreeAudioListener } from 'three'
+  import type { AudioListenerProps } from './AudioListener.svelte'
+
+  type $$Props = AudioListenerProps
+
+  export let position: $$Props['position'] = undefined
+  export let scale: $$Props['scale'] = undefined
+  export let rotation: $$Props['rotation'] = undefined
+  export let lookAt: $$Props['lookAt'] = undefined
+  export let viewportAware: $$Props['viewportAware'] = false
+  export let inViewport: $$Props['inViewport'] = false
+  export let castShadow: $$Props['castShadow'] = undefined
+  export let receiveShadow: $$Props['receiveShadow'] = undefined
+  export let frustumCulled: $$Props['frustumCulled'] = undefined
+  export let renderOrder: $$Props['renderOrder'] = undefined
+  export let visible: $$Props['visible'] = undefined
+  export let userData: $$Props['userData'] = undefined
+  export let dispose: $$Props['dispose'] = undefined
+
+  export let id: $$Props['id'] = undefined
+  export let masterVolume: $$Props['masterVolume'] = undefined
+
+  export const listener = new ThreeAudioListener()
+
+  export const context = listener.context
+  export const resumeContext = async () => await listener.context.resume()
+
+  $: if (masterVolume !== undefined) listener.setMasterVolume(masterVolume)
+
+  const { addAudioListener, removeAudioListener } = useThrelteAudio()
+
+  addAudioListener(listener, id)
+
+  onDestroy(() => {
+    removeAudioListener(id)
+  })
+</script>
+
+<Three
+  type={listener}
+  {position}
```

Comment:
Same here, please remove `{position}` et al and insert these via `{...$$restProps}` and remove these props from the script block